code/game/machinery/dna_infuser/organ_sets/carp_organs.dm

```diff
@@ -153,6 +153,7 @@
 	. = ..()
 	AddElement(/datum/element/noticable_organ, "%PRONOUN_Their skin has small patches of scales growing on it.", BODY_ZONE_CHEST)
 	AddElement(/datum/element/organ_set_bonus, /datum/status_effect/organ_set_bonus/carp)
+	AddElement(/datum/element/update_icon_blocker)
```

Comment:
Why does it break when the icon updates? Seems like a bandaid, or is this a GAGS quirk?

Reply:
When I did testing, it only happened with hearts from the infuser_organs.dmi file with the grayscale config set, so it might be. I also ran the Restart function and changed the beating variable because I thought it might be an error with a beating state sprite missing, no sprite updates and no errors. Tested with all other rat organs and no error there either. I went with this change because it matches the Goliath heart which has the same sprite and same element applied.